packages/client/hmi-client/src/components/workflow/tera-calibration-node-ciemss.vue

```diff
@@ -217,6 +217,17 @@ const runCalibrate = async () => {
 		paramsObj[d] = Math.random() * 0.05;
 	});
 
+	let start = state.timeSpan.start;
+	let end = state.timeSpan.end;
+	// If we have the min/max timestamp available from the csv asset use it
+	if (csvAsset.value) {
+		const tIndex = csvAsset.value.headers.indexOf('timestamp');
+		if (tIndex !== -1) {
+			start = csvAsset.value.stats?.[tIndex].minValue;
+			end = csvAsset.value.stats?.[tIndex].maxValue;
+		}
```

Comment:
Do we have guarantee that the that whatever "timestamp" is mapped to is a numeric value? Can they be, for example, date strings like "2012-02-02"? 

@Tom-Szendrey do you know?

Reply:
the `timestamp` mapping is just a column in a .csv
This technically can be any type

Reply:
@Jami159 @Tom-Szendrey this is marked as as resolved, do we already have a resolution here? Or do we know what to do for the next step?

Reply:
Talked offline - it depends on the stats parsing logic, which we will need to amend/fix 
https://github.com/DARPA-ASKEM/Terarium/blob/e98ed653cf560db9afd0b5af7f9a24faff3802a3/packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/DatasetResource.java#L305-L337